conformance/reports/v1.0.0/projectcontour-contour/README.md

```diff
@@ -5,5 +5,6 @@
 |API channel|Implementation version|Mode|Report|
 |-----------|----------------------|----|------|
 |x|[v1.28.1](https://github.com/projectcontour/contour/releases/tag/v1.28.1)|x|[link](./v1.28.1-report.yaml)|
+|x|[v1.28.2](https://github.com/projectcontour/contour/releases/tag/v1.28.2)|x|[link](./v1.28.2-report.yaml)|
 
 ## Reproduce
```

Comment:
I think we've started requiring reproduction steps for all future conformance reports, do you mind filling this out @sunjayBhatia?

Reply:
added!

Reply:
inspired by the format of the kong one